### PR TITLE
[CTS] Skip images and fill2D tests if feature is not supported

### PR DESCRIPTION
**Pull request by igchor**

### CODE REVIEW

**Review by pbalcer — APPROVED** [review]

**Comment by pbalcer** [comment]
Please run `make cppformat`.

File: test/conformance/testing/include/uur/fixtures.h [hunk 1]
```diff
@@ -372,17 +373,19 @@ struct urQueueTest : urContextTest {
...
     void SetUp() override {
         UUR_RETURN_ON_FATAL_FAILURE(urQueueTest::SetUp());
-        UUR_RETURN_ON_FATAL_FAILURE(
-            uur::KernelsEnvironment::instance->LoadSource("foo", il_binary));
-        ASSERT_SUCCESS(uur::KernelsEnvironment::instance->CreateProgram(
-            platform, context, device, *il_binary, nullptr, &program));
 
         size_t size = 0;
-        ASSERT_SUCCESS(urDeviceGetInfo(
+        UUR_ASSERT_SUCCESS_OR_UNSUPPORTED(urDeviceGetInfo(
```
Comment (aarongreig) [inline test/conformance/testing/include/uur/fixtures.h:378]: this isn't an optional query, adapters should return `false` if they don't support pipes

Comment (igchor) [reply test/conformance/testing/include/uur/fixtures.h]: Right, looks like only L0 didn't implement this. I added the missing query support.

**Comment by pbalcer** [comment]
@oneapi-src/unified-runtime-native-cpu-write please review

you need to rebase again :)

**Review by PietroGhg — APPROVED** [review]
Native CPU lgtm, sorry for the late review

**Comment by igchor** [comment]
> @oneapi-src/unified-runtime-native-cpu-write please review
> 
> you need to rebase again :)

Done.

**Comment by kbenzie** [comment]
@igchor there's a conflict needing fixed on `test/conformance/device/device_adapter_native_cpu.match`

**Comment by igchor** [comment]
> @igchor there's a conflict needing fixed on `test/conformance/device/device_adapter_native_cpu.match`

Rebased

**Comment by martygrant** [comment]
Pulled in by https://github.com/intel/llvm/pull/16410

**Pull request merged**